packages/react/src/date-picker/DatePickerInput.tsx

```diff
@@ -50,13 +51,6 @@ export const DateInput = forwardRef<HTMLInputElement, DateInputProps>(
 			borderBottomRightRadius: 0,
 		};
 
-		const ariaLabel = useMemo(() => {
```

Comment:
This has been moved into a prop, so date-picker and date-range-picker get generate their own custom aria-label